City-of-Helsinki/helsinki-paatokset
# PP-0: Fix Github dependabot warnings.

This _should_ fix these issues: https://github.com/City-of-Helsinki/helsinki-paatokset/security/dependabot

**To test**
* Checkout branch
* Run `npm i` in /public/modules/custom/hds_web_components
* Check that the HDS web components still work as intended
